docs/supported-metrics.md

```diff
@@ -54,3 +54,48 @@ In Mesmer we support 3 types of metrics:
 | Stream throughput            | counter |
 | Operator throughput          | counter |
 | Operator processing time     | counter |
+
+## ZIO (both Default-JVM-Metrics and others)
+
+| Metric                                | Type    |
+|---------------------------------------|---------|
+| Executor Capacity                     | gauge   |
+| Executor Concurrency                  | gauge   |
+| Executor Dequeued Count               |         |
+| Executor Enqueued Count               |         |
+| Executor Size                         | gauge   |
+| Executor Worker Count                 |         |
+| JVM Buffer Pool Capacity Bytes        | gauge   |
+| JVM Buffer Pool Used Buffers          | gauge   |
+| JVM Buffer Pool Used Bytes            | gauge   |
+| JVM Classes Loaded                    | gauge   |
+| JVM Classes Loaded Total              | gauge   |
+| JVM Classes Unloaded Total            | gauge   |
+| JVM GC Collection Seconds Count       |         |
+| JVM GC Collection Seconds Sum         |         |
+| JVM Info                              | gauge   |
+| JVM Memory Bytes Committed            | gauge   |
+| JVM Memory Bytes Init                 | gauge   |
+| JVM Memory Bytes Max                  | gauge   |
+| JVM Memory Bytes Used                 | gauge   |
+| JVM Memory Pool Allocated Bytes Total | counter |
+| JVM Memory Pool Bytes Committed       | gauge   |
+| JVM Memory Pool Bytes Init            | gauge   |
+| JVM Memory Pool Bytes Max             | gauge   |
+| JVM Memory Pool Bytes Used            | gauge   |
+| JVM Threads Current                   | gauge   |
+| JVM Threads Daemon                    | gauge   |
+| JVM Threads Deadlocked                | gauge   |
+| JVM Threads Deadlocked Monitor        | gauge   |
+| JVM Threads Peak                      | gauge   |
+| JVM Threads Started Total             | gauge   |
+| Null                                  | counter |
```

Comment:
Shouldn't it be `Forwarded Null` ?

Reply:
You are correct.

@jczuchnowski, I refactored the fix in this commit:
https://github.com/ScalaConsultants/mesmer/pull/470/commits/377f11eab460818b07249fe0b5c759f214d45413